bin/redshift/run_mvn_tests_on_aws.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
`run_mvn_tests_on_aws.sh` -- the "on_aws" feels redundant. where else could they run?

(actually, the tests are run "here")